documentation/symbols/SSQP_Key_Conventions.md

```diff
@@ -176,3 +176,32 @@ Example:
 **Version at pseudo-rva 0xFFFFFFFE:** `1`
 
 **Lookup key:** `system.private.corelib.ni.r2rmap/r2rmap-v1-f5fddf60efb0bee79ef02a19c3decba9/system.private.corelib.ni.r2rmap`
+
+
+### JavaScript Source Maps
+
+JavaScript source maps, which are used by browser developer tools to provide source-level debugging experiences, are [standardized](https://sourcemaps.info). In 
+Microsoft Edge, source maps can be indexed by the [SHA-256 hash of the script file they map](https://chromedevtools.github.io/devtools-protocol/tot/Debugger/#event-scriptParsed)
```

Comment:
This is a document about conventions. Not so much about a specific product. Is it necessary to name the particular browser?

Reply:
I don't particularly care one way or the next, but I reference the browser and the source of it. I don't believe that Windbg extensions are doing this yet, so it's a convention invented and used in one place, which makes it not really a "convention"?